Modules/CTP/include/CTP/CTPTrendingTask.h

```diff
@@ -62,16 +62,20 @@ class CTPTrendingTask : public PostProcessingInterface
-  const int mNumberOfInputs = 5;                                                                                                                                                                                                                                                                                                                                                         // number of thrended ctp inputs
-  int mClassIndex[5] = { 65, 65, 65, 65, 65 };                                                                                                                                                                                                                                                                                                                                           // indices of trended ctp classes, found in CTPconfig
-  int mInputIndex[5] = { 49, 49, 49, 49, 49 };                                                                                                                                                                                                                                                                                                                                           // indices of trended ctp inputs, found in CTPconfig
-  bool mCTPconfigFound = false;                                                                                                                                                                                                                                                                                                                                                          // bool telling whether the CTPconfing was already found
+  std::string mInputNamesDefault[5] = { "MTVX", "MVBA", "0DMC", "0EMC", "0PH0" }; // ctp inputs to be trended by default, unless modified in config.json
+  std::string mTrendedInputNames[5] = { "inputContentMinBias1:time", "inputContentMinBias2:time", "inputContentDMC:time", "inputContentEMC:time", "inputContentPHO:time" };
+  std::string mTrendedClassNames[5] = { "classContentMinBias1:time", "classContentMinBias2:time", "classContentDMC:time", "classContentEMC:time", "classContentPHO:time" };
+  std::string mTrendedInputRatioNames[4] = { "inputContentMinBias2/inputs", "inputContentDMC/inputs", "inputContentEMC/inputs", "inputContentPHO/inputs" };
+  std::string mTrendedClassRatioNames[4] = { "classContentMinBias2/classes", "classContentDMC/classes", "classContentEMC/classes", "classContentPHO/classes" };
+  std::string ctpinputs[49] = { "MT0A", "MT0C", "MTVX", "MTSC", "MTCE", "MVBA", "MVOR", "MVIR", "MVNC", "MVCH", "11", "12", "0UCE", "0DMC", "0USC", "0UVX", "0U0C", "0U0A", "COS", "LAS", "0EMC", "0PH0", "23", "24", "1ZED", "1ZNC", "1PHL", "1PHH", "1PHM", "30", "31", "32", "33", "34", "35", "36", "0EJ1", "0EG1", "0EJ2", "0EG2", "DJ1", "DG1", "DJ2", "DG2", "45", "46", "47", "EMBA", "49" }; // all ctp input names
```

Comment:
The inputs already exists in:
https://github.com/AliceO2Group/AliceO2/blob/bdde90491b05acd2505b67be723eda13033be949/DataFormats/Detectors/CTP/src/Configuration.cxx#L1071
Can not this beused ?